nuxeo-web-ui-ftest/test/pages/ui/browser.js

```diff
@@ -156,4 +157,32 @@ export default class Browser extends BasePage {
     });
   }
 
+  waitForNbChildren(nb) {
+    this.el.waitForVisible('nuxeo-data-table nuxeo-data-table-row');
+    driver.waitUntil(() => {
+      const rows = this.el.elements('nuxeo-data-table nuxeo-data-table-row');
+      let count = 0;
+      rows.value.forEach((row) => {
+        if (row.isVisible(`nuxeo-data-table-cell a.title`) && row.isVisible()) {
```

Comment:
The visibility check for the row should come before checking visibility for the row's children. And perhaps the name could be `waitForNbVisibleChildren`, since we're relying on visibility and the results might vary amongst machines with different resolutions and viewport sizes.

Reply:
>The visibility check for the row should come before checking visibility for the row's children. 
yes
>perhaps the name could be waitForNbVisibleChildren
not really, we check visibility because the table has an inner iron-list which has a dom node pool of size greater than the actual items list size. That's why we check visibility to have an actual count

Reply:
I agree, thus the suggestion of including "visible" in the method name, so it is clear for people using it and that do not know how iron list works, that it will only fetch visible children. But it is just nitpicking. If you keep this name, please add a comment or so stating that the results might vary with the viewport size.